Regression: 4.3.0 breaks ZopeTestCase

```python
.../site-packages/Testing/ZopeTestCase/__init__.py:54: in <module>
    from . import placeless  # NOQA
.../site-packages/Testing/ZopeTestCase/placeless.py:31: in <module>
    ContainerPlacelessSetup):
E   TypeError: Error when calling the metaclass bases
E       Cannot create a consistent method resolution
E   order (MRO) for bases PlacelessSetup, PlacelessSetup, PlacelessSetup, CleanUp
```

Seems to be the same problem like #30.

--- a/CHANGES.rst
+++ b/CHANGES.rst
@@ -5,7 +5,10 @@
 4.3.1 (unreleased)
 ==================
 
-- Nothing changed yet.
+- Make ``PlacelessSetup`` not extend ``zope.testing.cleanup.CleanUp``.
+  Extending ``CleanUp`` was introduced in 4.3.0 but turned out to have
+  unexpected consequences. See `issue 30
+  <https://github.com/zopefoundation/zope.i18n/issues/30>`_.
 
 
 4.3.0 (2017-12-18)

--- a/src/zope/i18n/testing.py
+++ b/src/zope/i18n/testing.py
@@ -19,7 +19,6 @@ This relies on :mod:`zope.publisher` being available.
 
 import re
 
-from zope.testing.cleanup import CleanUp
 from zope.testing import renormalizing
 
 rules = []
@@ -39,7 +38,7 @@ def setUp(test=None):
     zope.component.provideAdapter(BrowserLanguages)
 
 
-class PlacelessSetup(CleanUp):
+class PlacelessSetup(object):
 
     def setUp(self):
         """
@@ -56,5 +55,4 @@ class PlacelessSetup(CleanUp):
         <zope.publisher.browser.BrowserLanguages ...>
 
         """
-        super(PlacelessSetup, self).setUp()
         setUp()

